spark/v3.5/spark/src/test/java/org/apache/iceberg/TestFileIOSerialization.java

```diff
@@ -83,9 +83,7 @@ public void testHadoopFileIOKryoSerialization() throws IOException {
     FileIO deserializedIO = KryoHelpers.roundTripSerialize(serializableTable.io());
     Configuration actualConf = ((HadoopFileIO) deserializedIO).conf();
 
-    assertThat(toMap(actualConf)).as("Conf pairs must match").isEqualTo(toMap(expectedConf));
-    assertThat(actualConf.get("k1")).as("Conf values must be present").isEqualTo("v1");
-    assertThat(actualConf.get("k2")).as("Conf values must be present").isEqualTo("v2");
+    assertThat(actualConf).containsExactlyInAnyOrderElementsOf(expectedConf);
   }
```

Comment:
Thank you for your contribution! I have a question, why were these two `assertThat` removed?

Reply:
Thanks for the comment. I believe the new one checks if the key-value pairs match between `actualConf` and `expectedConf`, and also checks the existence of the entries `(k1, v1)` and `(k2, v2)`. If there's something missing, please let me know.

Reply:
@tomtongue Thanks for your reply! it makes sense to me.